go.mod

```diff
@@ -4,11 +4,16 @@ require (
 	github.com/beorn7/perks v1.0.1
 	github.com/cespare/xxhash/v2 v2.1.1
 	github.com/golang/protobuf v1.3.2
-	github.com/json-iterator/go v1.1.8
+	github.com/google/go-cmp v0.4.0 // indirect
```

Comment:
I'm not sure why about the indirect dependencies. I think they shouldn't be there.

Reply:
They got introduced by the update of dependencies. `go mod tidy` doesn't remove them.

It seems that's the way it has to be.

Do you know a way I could try to get rid of them?